app/Dao/UserDao.php

```diff
@@ -108,4 +99,33 @@ public function deleteUserById($id): void
         $user = User::findOrFail($id);
         $user->delete();
     }
+
+    /**
+     * Login user by email and password
+     * @param Request data
+     * @return User
+     */
```

Comment:
Please make sure function document is correct.
Check parameter type, name and response type.